src/lib/core/trigger.cc

```diff
@@ -50,3 +60,83 @@ trigger_run_reverse(struct rlist *list, void *event)
 			return -1;
 	return 0;
 }
+
+void
+trigger_fiber_run(struct rlist *list, void *event, double timeout)
```

Comment:
Eventually this function turns out to be unsued. I'd delete it at all

Reply:
fix